test_scripts/API/SetMediaClockTimer/ATF_HP_SetMediaClockTimer_With_Valid_enableSeek_Parameter.lua

```diff
@@ -0,0 +1,130 @@
+--------------------------------------------------------------------------------------------
```

Comment:
@OVikhrov, script crashes on start with:
Lua error:
./user_modules/shared_testcases/commonFunctions.lua:330: attempt to perform arithmetic on global 'NewTestSuiteNumber' (a nil value)